### ISSUE TITLE
Coracohumeral distance decreased

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11003]
**Preferred term label:**
Coracohumeral distance decreased 

**Synonyms**

**Definition (free text, please give PubMed ID)**
Axial view on shoulder X-ray or MRI showing decreased distance between posterior border of the coracoid process and anterior border of the humeral head of <6 mm; subcoracoid impingement; decreased coracohumeral interval

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Rotator cuff tear, Shoulder external impingement syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001455
- label: Decreased coracohumeral distance
- parents: [Abnormal shoulder morphology (HP:0003043)](https://hpo.jax.org/app/browse/term/HP:0003043)
- pmid: 
- def: Axial view on shoulder X-ray or MRI showing decreased distance between posterior border of the coracoid process and anterior border of the humeral head of less than 6 mm; subcoracoid impingement; decreased coracohumeral interval.
- comment: Decreased coracohumeral distance can be observed in conditions such as rotator cuff tear and shoulder external impingement syndrome.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11003
- synonyms: Coracohumeral distance decreased(NONE)
